egs/wsj/s5/utils/subword/prepare_lang_subword.sh

```diff
@@ -258,8 +276,15 @@ echo $ndisambig > $tmpdir/lex_ndisambig
 ( for n in `seq 0 $ndisambig`; do echo '#'$n; done ) >$dir/phones/disambig.txt
 
 # Create phone symbol table.
-echo "<eps>" | cat - $dir/phones/{silence,nonsilence,disambig}.txt | \
-  awk '{n=NR-1; print $1, n;}' > $dir/phones.txt
+if [[ ! -z $phone_symbol_table ]]; then
```

Comment:
You need to either make this single brackets or quote $phone_symbol_table.  This will crash if it's empty.

Reply:
I did some tests it seems it does not crash with empty variable...

Reply:
oh yes I see, I was testing the wrong way.  OK, will merge.